core/arch/arm/plat-stm/tz_a9init.S

```diff
@@ -111,12 +102,20 @@ _config_l2cc_r3p0:
 	 * Cacheable accesses have high prio (bit10=0)
 	 * Full Line Zero (FLZ) disabled (bit0=0)
 	 */
-	movw r1, #0x0800
-	movt r1, #0x3C48
+	movw r1, #(PL310_AUX_CTRL_INIT & 0xFFFF)
```

Comment:
with a macro the two lines could be reduced to something like:
`mov_imm r1, PL310_AUX_CTRL_INIT`
which is both easier to read and write in my opinion, an implementation in C is preferable of course.